src/index.html

```diff
@@ -22,6 +22,440 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="product-cards"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="img product"
+          class="product-cards__image"
+        />
+
+        <div class="product-cards__info">
+          <h2 class="product-cards__info-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="product-cards__info-product-code">Product code: 195434</p>
+        </div>
+
+        <div class="product-cards__raiting">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <span class="product-cards__raiting-rewiews">Reviews: 5</span>
+        </div>
+
+        <div class="product-cards__prise">
```

Comment:
```suggestion
        <div class="product-cards__price">
```
typo

Also, this comment wasn't fixed:
![image](https://github.com/user-attachments/assets/6872ef72-77d5-4125-99f9-dd9e87c163ba)